src/Client/AmplitudeClient.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace Acquia\Console\ContentHub\Client;
+
+use Consolidation\Config\Config;
+use Symfony\Component\Yaml\Yaml;
+use Zumba\Amplitude\Amplitude;
+
+/**
+ * Provides an amplitude client to log events.
+ */
+class AmplitudeClient {
+
+  /**
+   * Config directory to load Amplitude API Key YAML.
+   *
+   * @var array
+   */
+  protected $apiKeyDir = [
+    '/config',
+    'amplitude_api_key.yml',
+  ];
+
+  /**
+   * Amplitude API Key.
+   *
+   * @var string
+   */
+  protected $amplitudeApiKey = '';
+
+  /**
+   * The Zumba Amplitude client.
+   *
+   * @var \Zumba\Amplitude\Amplitude
+   */
+  protected $amplitude;
+
+  /**
+   * Constructs an instance of Amplitude Client.
+   *
+   * @param string $amplitude_user_id
+   *   The Amplitude user ID, usually the Content Hub Service
+   *   Client UUID which will be specific to subscription.
+   */
+  public function __construct(string $amplitude_user_id) {
+    $this->loadAmplitudeApiConfig();
+    $this->amplitude = (new Amplitude())->init($this->amplitudeApiKey, $amplitude_user_id);
+  }
+
+  /**
+   * Determines whether or not amplitude client is ready.
+   *
+   * @return bool
+   *   TRUE if Amplitude is ready or FALSE if not.
+   */
+  public function isReady(): bool {
```

Comment:
Is this function used anywhere? If not do we need it?

Reply:
Done.